docs/using-kontena/volumes.md

```diff
@@ -20,16 +20,18 @@ Volumes can be managed from the Kontena CLI using the `kontena volume xyz` comma
 
 ### Creating volumes
 
-`kontena volume create --driver rexray --scope instance my-volume`
+`kontena volume create --driver rexray/s3fs --scope instance my-volume`
 
 See volume [scopes](#volume-scopes) for details on the different scopes.
 
+The driver has to be specified **without** a version tag, i.e. `rexray/s3fs:latest` is **not** supported. This is because using a hard version info on volume specification will make running multiple volume plugin versions within the grid impossible and also makes upgrading volume plugins really hard. Kontena will match the reported plugin version automatically and thus is able to create and manage the volumes.
```

Comment:
Document the plugin version selection, and specifically, the case of having multiple plugin versions installed. Specifically, that you must only ever install one plugin version.